### ISSUE TITLE
Increase SU-57 Byelka Radar Range from 160-215nm, enable OECM ability for Himalayas EW Suite

### ISSUE THREAD

**Issue by HyperGopher** [PygmalionOfCyprus/cmo-db-requests#88]
### DB Selector

CWDB

### Affected DBID(s)

1821, 5447

### Summary of Changes

1. In game SU-57 has a max radar range of 160nm. But according to the designer from back in 2008, early AESA byelka radar prototype reportedly had a max range of 350-400km for an estimated 2.5m2 target (190-215nm), making it have a longer range than the hybrid PESA Irbis-E (160nm) and is alleged to be closer to the 220nm (400km) max range of the F-22 raptor.

"NIIP have publicly cited detection range performance of 350 to 400 km (190 to 215 NMI), which assuming a Russian industry standard 2.5m2 target, is also consistent with the 2008 model for an AESA radar using ~10W rated TR modules, which in turn is the power rating for the modules used in the Zhuk AE prototypes. This puts the net peak power at ~15 kiloWatts, slightly below the Irbis E, but even a very modest 25% increase in TR module output rating would overcome this."

With newer, higher power TR modules for later prototypes, the estimated range of the Byelka is estimated at 400km for 1.5m2 target (215nm) max range.

2. Also, the Himalayas ECM suite in game is only defensive. But similar to the F-35, it acts as an offensive EW in addition to defensive, so an option for OECM jamming would also more accurately reflect its abilities and be a great addition. It uses AESA arrays of the L-band and X-band radars, similar to an F-35 offensive EW system. This is consistent with active radar jamming. "The L402 Himaraya electronic countermeasures (ECM) suite made by the KNIRTI institute uses both its own arrays and that of the N036 radar."

### Sources

http://ausairpower.net/APA-2010-01.html

https://www.deagel.com/Protection%20Systems/Himalayas/a002985

https://military-history.fandom.com/wiki/Byelka_(radar)

**Comment by PygmalionOfCyprus** [comment]
I've tweaked the values on the Byelka to better return detection ranges of the kind indicated by various sources. Obviously we can't know for sure the sensor's capabilities, and the Su-57's capabilities are _routinely_ exaggerated. But with that said, following the lineage from the Irbis-E, it's not unreasonable to imagine a ~160nm range vs a ~1m2 target and a roughly ~215nm range against a ~2.5m2 target as described.

To achieve this I upped its max power to to 15kW (as indicated in your source) and fiddled with the gain. It's not going to be perfect -- at all -- on account of how little concrete info we have, but it should be closer now than it was before.

To be honest, I'm a little skeptical about some of these values. As it stands, this radar is going to slightly outperform even the F-35. But since the Su-57 is practically vaporware at the moment, I'm more comfortable plugging in some of these "prototype" values until we get a better feel for capabilities from deployments, combat performance, etc.

Also added OECM capability to the Himalayas, which is fair enough.

I'm going to double back to the Su-57/PAK-FA at some point. There are a lot of problems with it. But this is now two less.
